packages/clerk-js/src/ui/components/SignUp/__tests__/SignUpStart.test.tsx

```diff
@@ -230,12 +231,6 @@ describe('SignUpStart', () => {
         ),
       );
       expect(fixtures.signUp.create).toHaveBeenCalledWith({ strategy: 'ticket', ticket: 'test_ticket' });
-
-      expect(window.history.replaceState).toHaveBeenCalledWith(
-        undefined,
-        '',
-        expect.not.stringContaining('__clerk_ticket'),
-      );
```

Comment:
Why did we completely remove this test? 
Let's re-introduce it at the appropriate spot

Reply:
This whole test was added in the previous PR. The query param part of the test will have to be moved to a separate test.

Reply:
Added!